src/Symfony/Bridge/PhpUnit/bootstrap.php

```diff
@@ -12,6 +12,95 @@
+        'PHPUnit_Framework_Constraint_ExceptionMessageRegExp',
+        'PHPUnit_Framework_Constraint_FileExists',
+        'PHPUnit_Framework_Constraint_GreaterThan',
+        'PHPUnit_Framework_Constraint_IsAnything',
+        'PHPUnit_Framework_Constraint_IsEmpty',
+        'PHPUnit_Framework_Constraint_IsEqual',
+        'PHPUnit_Framework_Constraint_IsFalse',
+        'PHPUnit_Framework_Constraint_IsIdentical',
+        'PHPUnit_Framework_Constraint_IsInstanceOf',
+        'PHPUnit_Framework_Constraint_IsJson',
+        'PHPUnit_Framework_Constraint_IsNull',
+        'PHPUnit_Framework_Constraint_IsTrue',
+        'PHPUnit_Framework_Constraint_IsType',
+        'PHPUnit_Framework_Constraint_JsonMatches',
+        'PHPUnit_Framework_Constraint_JsonMatches_ErrorMessageProvider',
+        'PHPUnit_Framework_Constraint_LessThan',
+        'PHPUnit_Framework_Constraint_Not',
+        'PHPUnit_Framework_Constraint_ObjectHasAttribute',
+        'PHPUnit_Framework_Constraint_Or',
+        'PHPUnit_Framework_Constraint_PCREMatch',
+        'PHPUnit_Framework_Constraint_SameSize',
+        'PHPUnit_Framework_Constraint_StringContains',
+        'PHPUnit_Framework_Constraint_StringEndsWith',
+        'PHPUnit_Framework_Constraint_StringMatches',
+        'PHPUnit_Framework_Constraint_StringStartsWith',
+        'PHPUnit_Framework_Constraint_TraversableContains',
+        'PHPUnit_Framework_Constraint_TraversableContainsOnly',
+        'PHPUnit_Framework_Constraint_Xor',
+
+        'PHPUnit_Framework_Error',
+        'PHPUnit_Framework_Error_Deprecated',
+        'PHPUnit_Framework_Error_Notice',
+        'PHPUnit_Framework_Error_Warning',
+        'PHPUnit_Framework_Exception',
+        'PHPUnit_Framework_ExpectationFailedException',
+
+        'PHPUnit_Framework_MockObject_MockObject',
+
+        'PHPUnit_Framework_IncompleteTest',
+        'PHPUnit_Framework_IncompleteTestCase',
+        'PHPUnit_Framework_IncompleteTestError',
+        'PHPUnit_Framework_RiskyTest',
+        'PHPUnit_Framework_RiskyTestError',
+        'PHPUnit_Framework_SkippedTest',
+        'PHPUnit_Framework_SkippedTestCase',
+        'PHPUnit_Framework_SkippedTestError',
+        'PHPUnit_Framework_SkippedTestSuiteError',
+
+        'PHPUnit_Framework_SyntheticError',
+
+        'PHPUnit_Framework_Test',
+        'PHPUnit_Framework_TestCase', // override PhpUnit's ForwardCompat child class
+        'PHPUnit_Framework_TestFailure',
+        'PHPUnit_Framework_TestListener',
+        'PHPUnit_Framework_TestResult',
+        'PHPUnit_Framework_TestSuite', // override PhpUnit's ForwardCompat child class
+
+        'PHPUnit_Runner_BaseTestRunner',
+        'PHPUnit_Runner_Version',
+
+        'PHPUnit_Util_Blacklist',
+        'PHPUnit_Util_ErrorHandler',
+        'PHPUnit_Util_Test',
+        'PHPUnit_Util_XML',
+    ];
+    foreach ($classes as $class) {
+        class_alias($class, '\\'.strtr($class, ['_' => '\\']), true);
```

Comment:
Just wonder for a performance point of view (opcode). Is it better to remove variables? Ie.
```php
class_alias('A_Foo', '\A\Foo);
class_alias('A_Bar', '\A\Bar);
```

Reply:
`class_alias($class, strtr($class, '_', '\\'));` would be fine to me

Reply:
fixed `['_' => '\\']` replaced by `'_', '\\'`